cognite/client/_api/synthetic_time_series.py

```diff
@@ -46,28 +45,28 @@ def retrieve(
 
             Request a synthetic time series query with direct syntax
 
-                >>> from cognite.client.experimental import CogniteClient
+                >>> from cognite.client import CogniteClient
                 >>> c = CogniteClient()
-                >>> dps = c.datapoints.synthetic.retrieve(expression="TS{id:123} + TS{externalId:'abc'}", start="2w-ago", end="now")
+                >>> dps = c.synthetic_time_series.retrieve(expression="TS{id:123} + TS{externalId:'abc'}", start="2w-ago", end="now")
```

Comment:
I am not sure we can call it retrieve, since there might be "persistent synthetic time series", when you can save them and retrieve later.
maybe we should go for `query()` instead?
@sanderland thoughs?

Reply:
why was this moved at all? wtf?

Reply:
normal datapoints are client.datapoints.retrieve
persistent synthetic ts will be compatible and under the same endpoint
synthetic query was right where it belonged and there was no need to move it away from c.datapoints.synthetic

Reply:
@sanderland, got it, will move to `datapoints.synthetic`

Reply:
but...why was it moved in the first place? who did you discuss this with?

Reply:
revert to previous endpoint

Reply:
I disagree with ivan on query, see client.datapoints using retrieve, so this should match that endpoint as much as possible

Reply:
I don't agree with `retrieve` since retrieve is used for endpoints that can reference something by id.
query expressions is quite far from being just an id

Reply:
and I would reserve retrieve for persistent sts

Reply:
@polomani please see the datapoints api

Reply:
@hhenriques someone needs to break a tie

Reply:
@mrtraser Let's go with `datapoints.synthetic.query`. Also aligned with Time Lords and their opinion is also leaning to `retrieve` should be used for persistent sts

Reply:
done